src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -433,6 +485,9 @@ public Void call() throws Exception {
                     new OutputStreamWriter(this.output, VerboseProcess.UTF_8)
                 );
                 try {
+                    StringBuilder sb = new StringBuilder();
+                    String previousLine = null;
+                    int lineCount = 0;
```

Comment:
@dean-e-clark please read http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html